tasks/Ubuntu.yml

```diff
@@ -2,24 +2,29 @@
 # tasks file for osquery (Ubuntu specific)
 
 - name: Download osquery APT key
-  become: yes
   apt_key:
     keyserver: keyserver.ubuntu.com
     id: C9D8B80B
     state: present
   tags:
     - osquery
 
+- name: Ensure apt-transport-https is installed
+  apt:
+    name: apt-transport-https
+    state: present
+  when: ansible_distribution_release == 'xenial'
```

Comment:
No need to have a conditional checking if it's Xenial as it's ok to also ensure it's installed on Trusty.